apps/cricket-score-frontend/.env.example

```
@@ -0,0 +1,20 @@
+NEXT_PUBLIC_API_BASE_URL=https://cricketapi-r9zv.onrender.com
+
+# App Configuration
+NEXT_PUBLIC_APP_DESCRIPTION="Your ultimate source for real-time cricket scores and updates"
+NEXT_PUBLIC_APP_NAME=CricketScore
+
+# Feature Flags
+NEXT_PUBLIC_ENABLE_NOTIFICATIONS=false
+NEXT_PUBLIC_ENABLE_REAL_TIME=true
+
+# Environment
+NODE_ENV=development
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)